### ISSUE TITLE
I can not exit from IPython

### ISSUE THREAD

**Issue by diegozea** [tkf/IPython.jl#1]
Hi!
I tried it, but I can't exit from the IPython console after Ctrl+D. Tipping y and Enter doesn't work and Ctrl+C can't kill it.

![image](https://user-images.githubusercontent.com/2822757/39807650-1f76fb72-537d-11e8-81b0-14c57092dc78.png)

**Comment by tkf** [comment]
Hmm... Interesting. What is your `import IPython; IPython.__version__`? I just noticed that my IPython version is rather old (5.1.0) in the environment I'm testing. Upgrading...

**Comment by tkf** [comment]
It works in my laptop

```
julia> using IPython

julia>

In [1]: import IPython

In [2]: IPython.__version__
Out[2]: '6.3.1'

In [3]:
Do you really want to exit ([y]/n)?

julia> versioninfo()
Julia Version 0.6.2
Commit d386e40c17 (2017-12-13 18:08 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-4500U CPU @ 1.80GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)
```

**Comment by tkf** [comment]
What happens if you do `exit()`? It looks like IPython exits without asking anything in this case.

**Comment by tkf** [comment]
Oooops. I missed that you already fixed it in #3. Yes, I like #3!  For Python 2 support, it actually is better to use `from __future__ import print_function` since I may forget to put `print("")` instead of `print()`.  It's fixed in master.  Please check.

**Comment by diegozea** [comment]
Thanks! 
It works perfect in master now :) 
My IPython version is 5.4.1:
![imagen](https://user-images.githubusercontent.com/2822757/39918656-76275b16-5511-11e8-8f8e-69b292fdd1d0.png)
